README.md

```diff
@@ -277,9 +284,87 @@ Examples:
 ```
 
 
-Additional Projects:
-These projects are part of the SWAG family and either manipulate or utilize SWAG data.
+### Extended SWAG Schema (Version 2)
+The following describes the usage of all native fields included within the SWAG schema.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| schemaVersion | int | Describes the current schema version |
+| id | str | Unique ID of the account |
+| name | str | Canonical name, according to the account naming standard |
+| contacts | list(str) | List of team DLs that are majority stakeholders for the account |
+| provider | str | One of: AWS, GCP, Azure |
+| type | str | One of: Billing, Security, Shared Service, Service |
+| status | list(dict) | See status schema |
+| services | list(dict) | See service schema |
+| environment | str | One of: test, prod |
+| sensitive | bool |  Signifies if the account holds a special significance; (in scope for PCI, holds PII, contains sensitive key material, etc.,) |
+| description | str | Brief description about the account's intended use. |
+| owner | str | One of: <company-name>, AWS, Third-Party |
+| aliases | list(str) | List of other names this account may be referred to as |
+
+
+#### Service Schema
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| name | str | Name of the service |
+| regions | list(str) | List of regions - empty list indicates all regions |
+| roles | list(dict) | List of roles that control access to this service. See role schema |
+| metadata | dict | Service Level metadata
+
+
+#### Status Schema
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| region | str | Status per-region |
+| status | str | One of: Created, In-progress, Ready, Deprecated, In-active, Deleted |
+| notes | list(dict) | See notes schema |
+
+
+#### Notes Schema
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| date | date | Date note was created |
+| text | str | Free text field with additional information |
```

Comment:
Add a field for the user that made the note?

Reply:
We could... but I was mostly just document what is currently exists. If we want to add something to the schema we should open a separate issue.